<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Addoredit_Poruchenie</class>
 <widget class="QDialog" name="Addoredit_Poruchenie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Назначение поручения</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="5">
       <widget class="QLabel" name="label_error">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:red</string>
        </property>
        <property name="text">
         <string>Заполните обязательные поля</string>
        </property>
       </widget>
      </item>
      <item row="16" column="0" colspan="5">
       <widget class="QDateEdit" name="dateEdit">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="displayFormat">
         <string>dd MMMM yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="5">
       <widget class="QComboBox" name="comboBox_sotr">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="5">
       <widget class="QPlainTextEdit" name="plainTextEdit_text">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="5">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Введите описание поручения</string>
        </property>
       </widget>
      </item>
      <item row="15" column="0" colspan="5">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Выберите дату сдачи</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="5">
       <widget class="QLineEdit" name="lineEdit_title">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="5">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Введите заголовок</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="5">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Выберите исполнителя</string>
        </property>
       </widget>
      </item>
      <item row="19" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_ok">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ОК</string>
        </property>
        <property name="icon">
         <iconset resource="../resourses.qrc">
          <normaloff>:/img/img/ok.png</normaloff>:/img/img/ok.png</iconset>
        </property>
       </widget>
      </item>
      <item row="19" column="2">
       <widget class="QPushButton" name="pushButton_provereno">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Проверено</string>
        </property>
        <property name="icon">
         <iconset resource="../resourses.qrc">
          <normaloff>:/img/img/ok.png</normaloff>:/img/img/ok.png</iconset>
        </property>
       </widget>
      </item>
      <item row="19" column="3" colspan="2">
       <widget class="QPushButton" name="pushButton_cancel">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Отмена</string>
        </property>
        <property name="icon">
         <iconset resource="../resourses.qrc">
          <normaloff>:/img/img/cancel.png</normaloff>:/img/img/cancel.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit_file">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Выберите файл</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_choose">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Выбрать</string>
        </property>
        <property name="icon">
         <iconset resource="../resourses.qrc">
          <normaloff>:/img/img/1134892.png</normaloff>:/img/img/1134892.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>pushButton_choose</zorder>
     <zorder>label_5</zorder>
     <zorder>lineEdit_file</zorder>
     <zorder>label_5</zorder>
     <zorder>lineEdit_file</zorder>
     <zorder>pushButton_choose</zorder>
     <zorder>label_5</zorder>
     <zorder>lineEdit_file</zorder>
     <zorder>pushButton_choose</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_title</tabstop>
  <tabstop>plainTextEdit_text</tabstop>
  <tabstop>comboBox_sotr</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>pushButton_ok</tabstop>
 </tabstops>
 <resources>
  <include location="../resourses.qrc"/>
 </resources>
 <connections/>
</ui>
